### PR TITLE
OCM-6569 | fix: print role arn for hcp

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-6569

### CODE REVIEW

File: pkg/ocm/helpers.go [hunk 1]
```diff
@@ -869,6 +870,9 @@ func ValidateOperatorRolesMatchOidcProvider(reporter *reporter.Object, awsClient
 			return err
 		}
 		roleARN := *roleObject.Arn
+		if printInfo {
+			reporter.Infof("Using '%s'", *roleObject.Arn)
```
Comment (mikeshng) [inline pkg/ocm/helpers.go:874]: I think you moved this too far up. This print should only happen after it passes some validations.
I am thinking maybe before the `if hasManagedPolicies {` check.

**Review by mikeshng — APPROVED** [review]
/lgtm

**Review by robpblake — APPROVED** [review]
/lgtm

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**